src/ManagementCenter.md

```diff
@@ -151,6 +152,40 @@ Provide the details in this form for your LDAP server:
 
 Once configured, LDAP settings are saved in a file named `ldap.properties` under the `mancenter` folder mentioned in the previous section. If you want to update your settings afterwards, you need to update `ldap.properties` file and click "Reload Security Config" button on the login page.
  
+#### Password encryption
+ 
+By default, the password that you use in LDAP configuration is saved on the `ldap.properties` file in clear text. This might pose a security risk. To store the LDAP password in encrypted form, we offer the following two options:
+
+- **Provide a KeyStore password:** This will create and manage a Java KeyStore under the Management Center home directory. The LDAP password will be stored in this KeyStore in encrypted form.
+- **Configure an external Java KeyStore:** This will use an existing Java KeyStore. This option might also be used to store the password in an HSM that provides a Java KeyStore API.
+
+When you do either, the LDAP password you enter on the initial configuration UI dialog will be stored in encrypted form in a Java KeyStore instead of the `ldap.properties` file.
+
+##### Providing a master key for encryption
+
+There are two ways to provide a master key for encryption:
+
+- If you deploy Management Center on an application server, you need to set `MC_KEYSTORE_PASS` environment variable before starting Management Center. This option is less secure. You should clear the environment variable once you make sure you can log in with your LDAP credentials to minimize the security risk.
+- If you're starting Management Center from the commandline, you can start it with `-Dhazelcast.mc.askKeyStorePassword`. Management Center will ask for the KeyStore password upon start and use it as a password for the KeyStore it creates. This option is more secure as it only stores the KeyStore password in the memory.
+
+By default, Management Center will create a Java KeyStore file under the Management Center home directory with the name `mancenter.jceks`. You can change location of this file by using the `-Dhazelcast.mc.keyStore.path=/path/to/keyStore.jceks` JVM argument.
+ 
+##### Configuring an external Java KeyStore
+
+If you don't want Management Center to create a KeyStore for you and use an existing one that you've created before (or an HSM), set the following JVM arguments when starting Management Center:
+
+* `-Dhazelcast.mc.useExistingKeyStore=true`: Enables use of an existing KeyStore.
+* `-Dhazelcast.mc.existingKeyStore.path=/path/to/existing/keyStore.jceks`: Path to the KeyStore. Might be empty if using an HSM.
+* `-Dhazelcast.mc.existingKeyStore.pass=somepass`: Password for the KeyStore. Might be empty if HSM provides another means of unlocking HSM.
```

Comment:
...KeyStore. You do not have to set it if HSM provides another means to unlock HSM.